doc/src/devdocs/gc-sa.md

```diff
@@ -7,8 +7,18 @@ source code can be found in `src/clangsa`. Running it requires
 the clang dependency to be build. Set the `BUILD_LLVM_CLANG` variable
 in your Make.user in order to build an appropriate version of clang.
 You may also want to use the prebuilt binaries using the
-`USE_BINARYBUILDER_LLVM` options. Afterwards, running the analysis
-over the source tree is as simple as running `make -C src analyzegc`.
+`USE_BINARYBUILDER_LLVM` options.
+
+Alternatively (or if these do not suffice), try
+
+```sh
+make -C deps install-llvm install-clang install-llvm-tools install-libuv install-utf8proc install-unwind
```

Comment:
I believe `make -C src install-analysis-deps` also works, but it would be good if someone could confirm